app/assets/javascripts/admin.js

```diff
@@ -0,0 +1,2 @@
+//= require application
+//= require ck_classic_editor
```

Comment:
Do we need this at every admin action? If not then let's only include it where required